test/csuite/timestamp_abort/main.c

```diff
@@ -182,6 +184,14 @@ thread_ts_run(void *arg)
               "oldest_timestamp=%" PRIx64 ",stable_timestamp=%" PRIx64, ts, ts));
         testutil_check(conn->set_timestamp(conn, tscfg));
 
+        /*
+         * Only perform the reconfigure test every two seconds. If we do it too frequently then
+         * internal servers like the statistics server get destroyed and restarted too fast to do
+         * any work.
+         */
+        __wt_seconds((WT_SESSION_IMPL *)session, &now);
+        if (now - 2 <= last_reconfig)
```

Comment:
readability nit:  I think maybe this would be easier to understand:
```
if (now <= last_reconfig + 2)
   continue
```
or without the `continue`:
```
if (last_reconfig + 2 < now) {
   /* Set and reset ... */
}
```